README.md

```diff
@@ -1,7 +1,20 @@
 # docs-vim
-## Cloning Instructions
-This repository uses ``git submodule``, after cloning it you should run ``git
-submodule init`` and ``git submodule update``.
+## Installation
+```
+git clone https://github.com/matthewsot/docs-vim.git
+cd docs-vim
+git submodule init
+git submodule update
+```
+
+* Go to `chrome://extensions/` in Chrome or Chromium.
+* Enable "Developer Mode". (Right top of the screen)
+
+![image](https://user-images.githubusercontent.com/49868160/118260804-09b5ff00-b4b3-11eb-820a-58339ec95f33.png)
```

Comment:
I'm not super familiar with githubusercontent --- should these image files be inside this repository?

Reply:
I pasted the images in the README file and it automatically but the githubusercontent thing there. You can out them in the repository but it's not needed.